technology-considerations/droid_vs_siegfried.md

```diff
@@ -0,0 +1,22 @@
+## Droid vs Siegfried
+
+We need software to carry out the file format checks on the files uploaded to TDR. There are two choices, [Siegfried](https://github.com/richardlehane/siegfried) and [Droid](https://github.com/digital-preservation/droid)
+
+### Siegfried
+#### Advantages
+* Faster than Droid
+* Outputs results to json
+
+##### Disadvantages
+* Doesn't report multiple patterns correctly. There is as [issue](https://github.com/richardlehane/siegfried/issues/146) where the maintainer says they will fix it but we don't know when that will be.
+
+### Droid
+#### Advantages 
```

Comment:
Worth mentioning that DROID is a National Archives project, so it's easier for us to add new features?

Reply:
There is a line in there already that says we have more control over its development but I've put another line in the decision section as well.